### PR TITLE
fix(curriculum): Remove unnecessary assert message argument from English Front End Libraries challenges - 01

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to issue #36400

This PR removes assert message arguments from `testString` for the following sub sections of the `Front End Libraries` curriculum section:
* Bootstrap
* jQuery
* React and Redux
* Sass

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/react-and-redux/use-provider-to-connect-redux-to-react.english.md [hunk 1]
```diff
@@ -30,13 +30,13 @@ The code editor now shows all your Redux and React code from the past several ch
 ```yml
 tests:
   - text: The <code>AppWrapper</code> should render.
-    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(AppWrapper)); return mockedComponent.find('AppWrapper').length === 1; })(), 'The <code>AppWrapper</code> should render.');
+    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(AppWrapper)); return mockedComponent.find('AppWrapper').length === 1; })());
   - text: The <code>Provider</code> wrapper component should have a prop of <code>store</code> passed to it, equal to the Redux store.
     testString: getUserInput => assert((function() { const mockedComponent = Enzyme.mount(React.createElement(AppWrapper)); return getUserInput('index').replace(/\s/g,'').includes('<Providerstore={store}>'); })(), 'The <code>Provider</code> wrapper component should have a prop of <code>store</code> passed to it, equal to the Redux store.');
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/03-front-end-libraries/react-and-redux/use-provider-to-connect-redux-to-react.english.md]:
```suggestion
    testString: getUserInput => assert((function() { const mockedComponent = Enzyme.mount(React.createElement(AppWrapper)); return getUserInput('index').replace(/\s/g,'').includes('<Providerstore={store}>'); })());
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Just one slipped through.

**Review by thecodingaviator — COMMENTED** [review]
Things LGTM except what Oliver suggested above

**Comment by ojeytonwilliams** [comment]
Sorry about the extra space, should be okay now.

**Comment by RandellDawson** [comment]
I saw the extra space in one view, but in the view which shows all the changes, it did not show up, which is why I went ahead and made commit you requested.

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
@RandellDawson I just wish I knew what I did wrong in the first place.

Anyway, it all LGTM now.

**Pull request merged**